core/tests/puppeteer-acceptance-tests/utilities/user/logged-in-user.ts

```diff
@@ -47,6 +47,8 @@ const signUpUsernameField = 'input.e2e-test-username-input';
 const invalidEmailErrorContainer = '#mat-error-1';
 const invalidUsernameErrorContainer = '.oppia-warning-text';
 const optionText = '.mat-option-text';
+const profileDropdown = '.e2e-test-profile-dropdown';
+const learnerDashboardMenuLink = '.e2e-test-learner-dashboard-link';
```

Comment:
Maybe menu-link as well in the hyphenated name?

Reply:
Done!